openai/api_resources/abstract/api_resource.py

```diff
@@ -32,7 +33,7 @@ def class_url(cls):
         # with forward slashes (/), so replace the former with the latter.
         base = cls.OBJECT_NAME.replace(".", "/")  # type: ignore
         if cls.api_prefix:
-            return "/%s/%ss" % (cls.api_prefix, base)
+            return "/%s%ss" % (cls.api_prefix, base)
```

Comment:
Someone changed in on of the previous PRs the api_prefix to be "" which in turn causes the result to have "//" hence my change. I don't know if the previous change was meant to be permanent or temporary but for now, it only works like this.

Reply:
Not sure I understand this completely, do you have an example of what call breaks/causes the "//" issue?

I did a quick search of the repo, and I don't see anything setting api_prefix. AFAIK, your changes also only touches azure_api_prefix, so not sure why this case would be reached to begin with. Seems like it should skip this case and return url of `"/%ss" % (base)` instead.

If it does reach this case, did api_prefix and OBJECT_NAME have an extra "/"? Otherwise kind of confused what would cause it to have "//"

Reply:
api_prefix used to be "v1", a constant defined in this file above. It was changed in the following commit to "": https://github.com/sorinsuciu-msft/openai-python/commit/b39bddd99dc243bf09bec60f4866e2fb542d0e8f. When it was the constant "v1" it makes sense for the return to be: "/%s/%ss" % (cls.api_prefix, base) as it will return /v1/fine-tunes as the relative path for example. Now that it is just the empty string, it just returns "//fine-tunes" and hence the double /.

Reply:
It seems to work as expected now. As you said I might have had a strange OBJECT_NAME during my testing. Reverting.